_tests_/Itinerary.test.js

```diff
@@ -0,0 +1,15 @@
+/* globals describe it expect */
```

Comment:
This is here because of the 'describe is not defined' eslint error yes? You should only ever use rule exceptions like this in exceptional circumstances (pun intended!)
This is only happening because eslint doesnt know by default that this is a test file and so is looking for `describe` and `it` methods, which you have not defined, so it makes sense that it would complain about it. You can fix this by adding this to your `.eslintrc.json` file:
```
"env": {
  "jest": true
}
```

You can then remove this comment from the top of all your test files and all will work as expected